python/tests/test_lowlevel.py

```diff
@@ -2013,6 +2049,66 @@ def test_map_mutations_errors(self):
             self.assertRaises(_tskit.LibraryError, tree.map_mutations, genotypes)
 
 
+class TestTableMetadataSchema(unittest.TestCase, MetadataTestMixin):
+    def test_metadata_schema_attribute(self):
+        tables = _tskit.TableCollection(1.0)
+        for table in self.metadata_tables:
+            table = getattr(tables, f"{table}s")
+            # Check default value
+            self.assertEqual(table.metadata_schema, "")
+            # Set and read back
+            example = "An example of metadata schema with unicode 🎄🌳🌴🌲🎋"
```

Comment:
Just spotted that these are all different types of unicode tree. Nice!

Reply:
Tree-mendous. These little things keep me sane.